3_ProtectionProfile/PP config.adoc

```diff
@@ -0,0 +1,121 @@
+= PP-Configuration for Protection Profile for Mobile Device Fundamentals and collaborative Protection Profile Module for Mobile biometric enrolment and verification - for unlocking the device -
+:showtitle:
+:toc:
+:table-caption: Table
+:revnumber: 0.8
+:revdate: 2019-01-31
+
+== Acknowledgements
+
+This PP-Configuration was developed by the Biometrics Security (BS) international Technical Community (iTC) with representatives from industry, Government agencies, Common Criteria Test Laboratories, and members of academia.
+
+:sectnums:
+
+== Introduction
+
+=== Overview
+
+The purpose of a PP-Configuration is to define a Target of Evaluation (TOE) that combines Protection Profiles (PPs) and PP-Modules for various technology types into a single configuration that can be evaluated as a whole. The scope includes the definition of the configuration of a mobile device that has biometric enrolment and verification capability. The TOE will be defined by a combination of the components described in section 1.3.
+
+=== PP-Configuration Reference
+
+This PP-Configuration is identified as follows:
+
+* PP-Configuration for Protection Profile for Mobile Device Fundamentals and collaborative Protection Profile Module for Mobile biometric enrolment and verification - for unlocking the device - version 0.8, January 31, 2019
+* As a shorthand reference, it can be identified as "BIO+MDF"
+
+=== PP-Configuration Components Statement
+
+This PP-Configuration includes the following components:
+
+* Base-PP: Protection Profile for Mobile Device Fundamentals [MDFPP]
+* PP-Module: collaborative Protection Profile Module for Mobile biometric enrolment and verification - for unlocking the device - [BIOcPP Module].
+
+== Conformance Claims
+
+=== CC Conformance
+
+*Conformance Statement*
+
+To be conformant to this PP-Configuration, an ST must demonstrate Exact Conformance, a subset of Strict Conformance as defined in [CC1] (ASE_CCL).
```

Comment:
Sadly, neither [CC1} Not the addenda define the term "Exact Conformance".
Noting that many of the cPP "get around" this issue by defining it within the cPP.
The candidate MDF PP V3.2 is incorrect with the statement "An ST must claim exact conformance to this PP, as defined in the CC and CEM addenda for Exact Conformance, Selection-Based SFRs, and Optional SFRs (dated May 2017)." 

So I would suggest that this PP-Module avoid the issue for now and change the text to:
"To be conformant to this PP-Configuration, an ST must demonstrate Exact Conformance, a subset of Strict Conformance as defined in [MDFPP]."